src/main/java/org/opensearch/security/ssl/util/ExceptionUtils.java

```diff
@@ -76,7 +76,7 @@ public static OpenSearchException createJwkCreationException(Throwable cause) {
         return new OpenSearchException("An error occurred during the creation of Jwk: {}", cause, cause.getMessage());
     }
 
-    public static OpenSearchException createTransportClientNoLongerSupportedException() {
-        return new OpenSearchException("Transport client authentication no longer supported.");
+    public static OpenSearchException clusterWrongNodeCertConfigException() {
+        return new OpenSearchException("Node certificate configuration is wrong or certificate is invalid.");
```

Comment:
nit: wdyt about including the `principal` in the error message as well.

Something like:

`Node presenting certificate with SSL Principal {principal} could not securely connect to the cluster. Please ensure the principal is correct and present in the nodes_dn list`

`nodes_dn` list is a list of trusted distinguished names. This could either be statically defined in `opensearch.yml` or dynamic through the security index with the `nodes.yml` config file.

Reply:
Yes, it's better. I updated the eception message.